### PR TITLE
247: Support atomic pushes in with /integrate command

### PR DESCRIPTION
**Pull request by rwestberg**
Hi all,

Please review this change that allows pushing without automatic rebasing.

Best regards,
Robin

## Progress
- [x] Change must not contain extraneous whitespace
- [x] Change must be properly reviewed

## Issue
[SKARA-247](https://bugs.openjdk.java.net/browse/SKARA-247): Support atomic pushes in with /integrate command

## Approvers
 * Erik Helin ([ehelin](@edvbld) - **Reviewer**) **Note!** Review applies to 7497ada979f17fa63e2643ed2e4cf93ee54db6a1

### CODE REVIEW

File: bots/pr/src/main/java/org/openjdk/skara/bots/pr/IntegrateCommand.java [hunk 1]
```diff
@@ -108,6 +121,9 @@ public void handle(PullRequestBot bot, PullRequest pr, CensusInstance censusInst
                 if (!ProjectPermissions.mayCommit(censusInstance, pr.author())) {
                     reply.println(ReadyForSponsorTracker.addIntegrationMarker(pr.headHash()));
                     reply.println("Your change (at version " + pr.headHash() + ") is now ready to be sponsored by a Committer.");
+                    if (!args.isBlank()) {
+                        reply.println("Note that your sponsor will make the final decision on which target hash to integrate on.");
```
Comment (edvbld) [inline bots/pr/src/main/java/org/openjdk/skara/bots/pr/IntegrateCommand.java]:
```suggestion
                        reply.println("Note that your sponsor will make the final decision on which target hash to integrate upon.");
```

Comment (rwestberg) [reply bots/pr/src/main/java/org/openjdk/skara/bots/pr/IntegrateCommand.java]: How about onto, as in `git rebase --onto`?

Comment (edvbld) [reply bots/pr/src/main/java/org/openjdk/skara/bots/pr/IntegrateCommand.java]: :+1:

**Pull request closed without merging**